Formula/jenv.rb

```diff
@@ -15,12 +15,24 @@ def install
   end
 
   def caveats
-    <<~EOS
-      To activate jenv, add the following to your #{shell_profile}:
+    if preferred == :fish
+      s = <<~EOS
+        To activate jenv, run the following commands:
 
-        export PATH="$HOME/.jenv/bin:$PATH"
-        eval "$(jenv init -)"
-    EOS
+      EOS
+      s << "  fish_add_path #{bin}\n" if bin.directory?
```

Comment:
Why is this needed? `jenv` is not keg-only, so it should already be in a user's path.

Reply:
ah thanks, removed